draft-rfcs/working-group-terminology.md

```diff
@@ -2,96 +2,274 @@
-- Groups have at least one shepherd who acts as the leader of the group and is responsible for writing the initial charter, handling administrative and communication tasks, as well as delegating responsibilities to other members in the group.
-- Groups should have a liaison member associated with an official Rust team.
-  - This liaison may also be but is not required to be a shepherd.
-- Membership for both groups are represented on the official rust-lang.org website.
-- Membership requirements for both groups is decided by the shepherd and should be stated in the charter.
-  - Initial membership should try to represent people who have been participating regularly and productively in the respective area.
-- Neither group has _"formal decision making power"_. Where "formal decision making power" is defined as being able to accept RFCs on `rust-lang/rfcs`. Similarly, neither group has representation on the Core team.
-  - Groups are of course able to create RFCs as well as advocate their concerns and desired changes to the Rust teams and community.
-- Both groups are entitled to spaces in any or all of Rust's officially managed discussion forums.
   - As the time of this writing this includes [Zulip] and [Discord].
-  - Groups are also free to create spaces on platforms where the Rust project does not have a official presence. Moderation for these spaces should still follow Rust's [Code of Conduct].
+  - Groups are also free to create spaces on platforms where the Rust project
+    does not have a official presence. Moderation for these spaces should still
+    follow Rust's [Code of Conduct].
 
+[rust-lang.org]: https://rust-lang.org
 [zulip]: https://rust-lang.zulipchat.com
 [discord]: https://discord.gg/rust-lang
 [code of conduct]: https://www.rust-lang.org/policies/code-of-conduct
 
-### Working Groups
+### Initial Setup
 
-Working Groups typically focus on a specific domain of expertise that is not covered by an existing Rust team. Examples of this include Embedded, WebAssembly, and Game Development.
+Once a group has been approved, a pull request with the initial set of members
+should be made to [`rust-lang/team`]. Please refer to team's documentation for
+how to create a group.
 
-Working groups are a way of finding new domains that would benefit from continual development. Though it is not required, if a working group has demonstrated consistent productivity over a significant period time, it can become a Rust team.
+It is recommended for the working group to create a `wg-{name}`/`project-{name}`
+repository under the `rust-lang` organisation using the [project group
+template], and making any relevant changes and personalisation.
 
-- Creation of a Working Group is approved by the Core team.
+[`rust-lang/team`]: https://github.com/rust-lang/team
+[project group template]: https://github.com/rust-lang/project-group-template
```

Comment:
Will working groups and project groups share a template? (I don't see a specific reason why not, but I want to make sure that's the intention.)

Reply:
Yeah that is my intention currently.